prac_09/silver_service_taxi_test.py

```diff
@@ -0,0 +1,12 @@
+"""
+testing Silver Service Taxi Class.
+"""
+
+
+from silver_service_taxi import SilverServiceTaxi
+
+silver_service_taxi = SilverServiceTaxi("Black & White", 100, fanciness=1.5)
```

Comment:
It's optional but you could do some more testing for cars of different fanciness (i guess that way you know the additional fare for fancier cars works)